### PR TITLE
Fix strange animation to the left when unminimizing windows

### PR DESCRIPTION
**Pull request by mariospr**
https://phabricator.endlessm.com/T19659

### CODE REVIEW

File: js/ui/windowManager.js [hunk 1]
```diff
@@ -1579,8 +1579,10 @@ const WindowManager = new Lang.Class({
                          });
         } else {
             let monitor = Main.layoutManager.monitors[actor.meta_window.get_monitor()];
-            let xDest = monitor.x + actor.x;
```
Comment (rshuler) [inline js/ui/windowManager.js]: The removal of xDest and yDest should be moved to the next commit.

Comment (mariospr) [reply js/ui/windowManager.js]: Oops! Late-evening rebasing after long days is not a good idea :). Fixed

File: js/ui/windowManager.js [hunk 2]
```diff
@@ -1641,14 +1642,19 @@ const WindowManager = new Lang.Class({
                          });
         } else {
             let monitor = Main.layoutManager.monitors[actor.meta_window.get_monitor()];
+            if (!monitor) {
+                this._unminimizeWindowDone(shellwm, actor);
+                return;
+            }
```
Comment (smspillaz) [inline js/ui/windowManager.js]: This strikes me as a little strange - `monitor` would only be `null` in cases where the window didn't have a valid monitor. Is this a common case?

Comment (mariospr) [reply js/ui/windowManager.js]: I believe it can also be null if you don't have any monitor connected at the time.

**Pull request merged**